tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -79,8 +79,18 @@ def operation(self, diff_entry):
             # Ignore change in metadata for now, they have no impact
             return
 
-        # So method signaure changed. Be vague for now
-        self.breaking_changes.append(_SIGNATURE_CHANGE.format(operation_name, function_name))
+        if remaining_path[0] == "parameters":
+            old_parameters_list = self._old_report["operations"][operation_name]["functions"][function_name]['parameters']
+            new_parameters_list = self._new_report["operations"][operation_name]["functions"][function_name]['parameters']
+            old_parameters = {param_name['name'] for param_name in old_parameters_list}
+            new_parameters = {param_name['name'] for param_name in new_parameters_list}
+            # The new parameter is optional or not. Be breaking change for now.
+            for removed_parameter in old_parameters - new_parameters:
+                self.breaking_changes.append(_REMOVE_OPERATION_PARAM.format(operation_name, function_name, removed_parameter))
+            for added_parameter in new_parameters - old_parameters:
+                self.breaking_changes.append(_ADD_OPERATION_PARAM.format(operation_name, function_name,added_parameter))
+            return
+        raise NotImplementedError("Other situations. Be err for now.")
```

Comment:
```suggestion
        raise NotImplementedError(f"Other situations. Be err for now: {str(remaining_path)}")
```
I think it is better ton contain details in exception info